### ISSUE TITLE
Can't add Image to Excel file

### ISSUE THREAD

**Issue by mobilestar0223** [renebigot/XlsxReaderWriter#45]
I have used this project to create xlsx file and make good excel file.
Most things work well, but I can't add image to the file.
I wrote code like sample but I can't see image in the xlsx file.
Here is my code that I used.

let image = UIImage(named: "Image")
let drawing: BRAWorksheetDrawing = workingWorkSheet.addImage(image, betweenCellsReferenced: "C7", and: "F25", withInsets: UIEdgeInsetsZero, preserveTransparency: false)
drawing.insets = UIEdgeInsetsMake(0.0, 0.0, 0.5, 0.5)

**Comment by renebigot** [comment]
Is any other modification saved ? Like if you put "Hello World" in A1, can you read it with excel ?

**Comment by mobilestar0223** [comment]
Yes, I can add and read text in excel file. But I can't add picture.
I have used your sample excel file and deleted whole content and then try again. The result is same.
But if I put one image into sample file and use it, I can add picture to the file.
Do you know this reason?
Please let me know why and how to solve this problem.
Thanks.

**Comment by shoaib-akhtar** [comment]
Any progress on this? I am facing same issue. Unable to add image in worksheet.

**Comment by jmcusac** [comment]
Add an image to the excel worksheet. It can be a small "white" image in a location you wouldn't normally have any data attached to.

Once an image is previously in sheet, the dropped images from this git will insert properly. I dealt with this issue at length myself.

**Comment by developertechnostacks** [comment]
@jmcusac 

if you have solved this earlier can you help me out, as i am facing similar issue .

**Comment by Aashiii-12** [comment]
I have used your library to create xlsx file and make good excel file.
Most things work well, but I can't add image to the file.
I wrote code like sample but I can't see image in the xlsx file.
I already put one image into sample file and use it, still I can't add any Image programatically in my xlsx file.
Here is my code that I used.

```
NSString *documentPath = [[NSBundle mainBundle] pathForResource:@"testWorkbook" ofType:@"xlsx"];
BRAOfficeDocumentPackage *spreadsheet = [BRAOfficeDocumentPackage open:documentPath];

BRAWorksheet *fooWorksheet = [spreadsheet.workbook createWorksheetNamed:@"Foo"];
BRAWorksheet *worksheet = [spreadsheet.workbook createWorksheetNamed:@"Foo" byCopyingWorksheet:worksheetToCopy];
[[fooWorksheet cellForCellReference:@"B2" shouldCreate:YES] setStringValue:@"FOO / BAR"];
    
UIImage *image = [UIImage imageNamed:@"123.jpeg"];
BRAWorksheetDrawing *drawing = [fooWorksheet addImage:image betweenCellsReferenced:@"A1" and:@"D10"withInsets:UIEdgeInsetsZero preserveTransparency:NO];

drawing.insets = UIEdgeInsetsMake(0., 0., .5, .5);
    
[spreadsheet save];
[spreadsheet.workbook save];

NSString *fullPath = [[NSSearchPathForDirectoriesInDomains(NSDocumentDirectory, NSUserDomainMask, YES) lastObject] stringByAppendingPathComponent:@"workbookCopy.xlsx"];
[spreadsheet saveAs:fullPath];
NSLog(@"%@",fullPath);
```

**If you have solved this earlier can you help me out asap.**
@renebigot
